dat.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
What is that?

Reply:
i put the hc project into a dat project for testing purposes .. dat.json is a file that has to be in the root folder. 
i took it out